content/2_Bronze/Intro_Graphs.mdx

```diff
@@ -2,7 +2,7 @@
 id: intro-graphs
 title: Introduction to Graphs
 author: Darren Yao, Benjamin Qi
-contributors: Nathan Gong
+contributors: Nathan Gong, Ryan Chou, Neo Wang, Aadit Ambadkar, Sandeep Wadhwa, Kevin Sheng, Nikhil Chatterjee
```

Comment:
i'm ngl idt neo, sandeep, or aadit have anything in the final sol :skull:

Reply:
okay LMAO